charts/juicefs-csi-driver/templates/controller.yaml

```
@@ -19,6 +19,13 @@ spec:
       labels:
         app: juicefs-csi-controller
         {{- include "juicefs-csi.selectorLabels" . | nindent 8 }}
+        {{- if .Values.controller.extraLabels }}
+        {{- toYaml .Values.controller.extraLabels | nindent 8}}
+        {{- end}}
+      {{- with .Values.controller.extraAnnotations }}
+      annotations:
+          {{- toYaml . | nindent 8 }}
```

```suggestion
      {{- toYaml . | nindent 8 }}
```

plz format like this